gwcs/coordinate_frames.py

```diff
@@ -168,9 +167,15 @@ def coordinates(self, *args):
         """ Create world coordinates object"""
         raise NotImplementedError("Subclasses may implement this")
 
+    def coordinate_to_quantity(self, *coords):
+        """
+        Given a rich coordinate object return an astropy quantity object.
+        """
+        # NoOp leaves it to the model to handle
+        return coords
```

Comment:
Perhaps this should return a `NotImplementedError`. Otherwise it's misleading in the case when a CoordinateFrame class does not implement it and it returns numbers.

Also a general question: How does this relate to the `coordinates` method, just above it but also in various CoordintaeFrame classes? In some cases the implementation is identical.

Reply:
The relationship to the `coordinates` method, is that it's basically the inverse. The coordinates method takes the output of the transform and constructs a rich coordinate object. The `coordinate_to_quantity` method, takes the rich coordinate object as input and constructs a quantity to pass into the model.

Reply:
I *think* the reason why I implemented this a `return coords` is to support arbitrary frames with for weird models like the lookup table one in #132 I would think that the transform will throw an error if this fails, which I guess isn't as friendly as throwing it here... :thinking: